lib/plugin.js

```diff
@@ -56,6 +57,8 @@ module.exports = function (wct, pluginOptions) {
     collection.att('passed', stats.passing);
     collection.att('failed', stats.failing);
     collection.att('skipped', stats.pending);
+    collection.att('total', stats.passing + stats.failing + stat.pending);

```

Comment:
shouldn't it be `stats.pending` here rather than `stat.pending` ?

Reply:
correct this is a typeo, 
thank you for finding it, i did not notice